yt_dlp/extractor/youtube/_video.py

```diff
@@ -3541,6 +3544,36 @@ def _download_initial_webpage(self, webpage_url, webpage_client, video_id):
                 }))
         return webpage
 
+    def _parse_instream_ad_renderer(self, renderer):
+        instream_renderer = traverse_obj(renderer, ('instreamVideoAdRenderer', {dict})) or {}
+        length_ms = traverse_obj(instream_renderer, ('skipOffsetMilliseconds', {int}))
+        if length_ms is not None:
+            self.write_debug(f'Detected a {length_ms}ms skippable ad')
+            return length_ms
+        length_ms = traverse_obj(instream_renderer, ('playerVars', {urllib.parse.parse_qs}, 'length_seconds', -1, {int_or_none(invscale=1000)}))
+        if length_ms is not None:
+            self.write_debug(f'Detected a {length_ms}ms non-skippable ad')
```

Comment:
Same here, but `unskippable`
```suggestion
            length = length_ms / 1000 if length_ms % 1000 else length_ms // 1000
            self.write_debug(f'Detected a {length}s unskippable ad')
```